### PR TITLE
Fix Node.js examples and readme

### PR DESCRIPTION
**Pull request by kriskowal**
Since we moved to using ready-signal instead of after.

### CODE REVIEW

**Comment by kriskowal** [comment]
review @raynos @jcorbin

**Comment by jcorbin** [comment]
:+1:

File: node/README.md [hunk 1]
```diff
@@ -16,12 +16,13 @@ Status is being tracked in #78.
...
 ```js
 var TChannel = require('tchannel');
-var after = require('after');
+var CountedReadySignal = require('ready-signal/counted');
 
 var server = new TChannel();
 var client = new TChannel();
 
-var listening = after(2, function (err) {
+var ready = CountedReadySignal(3);
```
Comment (Raynos) [inline node/README.md]: why 3 ? should that be two ?

**Comment by Raynos** [comment]
other then the 2/3 typo, lgtm.

**Comment by kriskowal** [comment]
Fixed signal error in README

**Pull request merged**